Plugins/SwiftLintPlugin/SwiftLintPluginError.swift

```diff
@@ -0,0 +1,18 @@
+import PackagePlugin
+
+enum SwiftLintPluginError: Error, CustomStringConvertible {
+    case pathNotInDirectory(path: Path, directory: Path)
+    case swiftFilesNotInProjectDirectory(Path)
+    case swiftFilesNotInWorkingDirectory(Path)
+
+    var description: String {
+        switch self {
+        case let .pathNotInDirectory(path, directory):
+            return "Path '\(path)' is not in directory '\(directory)'."
+        case let .swiftFilesNotInProjectDirectory(directory):
+            return "Swift files are not in project directory '\(directory)'."
+        case let .swiftFilesNotInWorkingDirectory(directory):
+            return "Swift files are not in working directory '\(directory)'."
+        }
```

Comment:
You may use a `switch` expression:
```suggestion
        switch self {
        case let .pathNotInDirectory(path, directory):
            "Path '\(path)' is not in directory '\(directory)'."
        case let .swiftFilesNotInProjectDirectory(directory):
            "Swift files are not in project directory '\(directory)'."
        case let .swiftFilesNotInWorkingDirectory(directory):
            "Swift files are not in working directory '\(directory)'."
        }
```